stanovy_vpsFree.cz.md

```diff
@@ -154,149 +166,200 @@ informačním systému spolku
 5. Rada Spolku je povinna do programu Schůze zahrnout body, které navrhne
 Kontrolní komise a jakýkoliv člen Spolku
 
-### §11 Jednání
+### §11 Jednání schůze členů
 
 1. Jednání probíhá podle programu, který přítomní členové schválí na začátku
 Schůze
-2. Řádně svolaná Schůze členů je vždy usnášeníschopná. Schůze členů rozhoduje 
-prostou většinou hlasů přítomných členů.
+2. Řádně svolaná Schůze členů je usnášeníschopná při účasti alespoň třetiny
+členů. Schůze členů rozhoduje prostou většinou hlasů přítomných členů.
```

Comment:
Je to v pořádku - http://poradna.crdm.cz/clenska-schuze/usnasenischopnost-clenske-schuze-740 .